### PR TITLE
New parameter in the random-report-gen.gmp script

### PR DESCRIPTION
**Pull request by y0urself**
New parameter in the random-report-gen.gmp script by which we can add not_vuln host_details to hosts

**Checklist**:

- [ ] Tests
- [x] [CHANGELOG](https://github.com/greenbone/gvm-tools/blob/master/CHANGELOG.md) Entry
- [ ] Documentation

### CODE REVIEW

File: scripts/random-report-gen.gmp [hunk 1]
```diff
@@ -226,11 +226,17 @@ def generate_host_detail_elem(
     return host_detail_elem
 
 
-def generate_additional_host_details(n_host_details, host_details):
+def generate_additional_host_details(n_details, host_details, not_vuln):
```
Comment (bjoernricks) [inline scripts/random-report-gen.gmp]:
```suggestion
def generate_additional_host_details(n_details, host_details, *, not_vuln=False):
```

File: scripts/random-report-gen.gmp [hunk 2]
```diff
@@ -303,7 +316,15 @@ def generate_host_elem(
     if n_host_details:
         host_elem.extend(
             generate_additional_host_details(
-                n_host_details, data["host_details"]
+                n_host_details, data["host_details"], False
```
Comment (bjoernricks) [inline scripts/random-report-gen.gmp]:
```suggestion
                n_host_details, data["host_details"], not_vuln=False
```

File: scripts/random-report-gen.gmp [hunk 3]
```diff
@@ -303,7 +316,15 @@ def generate_host_elem(
...
+            )
+        )
+
+    dev = n_not_vuln / 10
+    if n_not_vuln:
+        host_elem.extend(
+            generate_additional_host_details(
+                n_not_vuln + randrange(-dev, dev), data["not_vuln"], True
```
Comment (bjoernricks) [inline scripts/random-report-gen.gmp]:
```suggestion
                n_not_vuln + randrange(-dev, dev), data["not_vuln"], not_vuln=True
```

**Comment by y0urself** [comment]
This should work now. #229 
Thanks for finding this bug.

**Pull request merged**